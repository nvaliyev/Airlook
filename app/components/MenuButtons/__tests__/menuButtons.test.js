import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import DrawerButtons from '..';
describe(' DrawerButtons', () => {
    test(' DrawerButtons snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(< DrawerButtons/>);
        expect(snapshot).toMatchSnapshot();
    });
});


