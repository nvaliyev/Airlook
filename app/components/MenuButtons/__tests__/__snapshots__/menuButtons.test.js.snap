// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` DrawerButtons  DrawerButtons snapshot 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  style={
    Object {
      "borderBottomColor": "#6f347c",
      "flexDirection": "row",
      "height": 65,
      "justifyContent": "flex-start",
      "marginHorizontal": 5,
      "padding": 15,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 18,
        "marginHorizontal": 30,
      }
    }
  />
</TouchableOpacity>
`;
